### ISSUE TITLE
All API Projects need a strict API path naming convention

### ISSUE THREAD

**Issue by Greenheart** [FreeCodeCamp/CurriculumExpansion#40]
## Why?
* Easier testing --> leads to more accurate test results and better feedback
* Teach a good naming convention for APIs - `example.com/api/<api-version>/<some-api-route`
  * API version isn't necessary for FCC  - But it could be useful to explain this naming convention before pointing out that FCC doesn't require it for these projects.

**Comment by Em-Ant** [comment]
I definitely agree with @Greenheart . 
Having a version field seems too much to me, for our challenges.

@QuincyLarson if it's OK for you, We'll close this and update the API projects endpoints to be conforming to the pattern `example.com/api/<some-route>`

**Comment by alayek** [comment]
At least we should mention that versioning is typically used, and is a good practice.

**Comment by QuincyLarson** [comment]
@Em-Ant @Greenheart @alayek yes - our goal is to get as much of the test logic as possible into Free Code Camp's seed directory (and as little as absolutely necessary remaining on HyperDev) which should reduce the need for new versions of the HyperDev repos.
